Automattic/woocommerce-payments
# Link the Requirements Component with the form

Fixes #4037 

#### Changes proposed in this Pull Request









#### Testing instructions





* Add `return true;` to the `is_in_onboarding_treatment_mode()` function in `class-wc-payments-utils.php`.
* Make the plugin act as if in onboarding mode and disconnected via the WCPay Dev settings.
* Go to `Payments`, click the connect button to redirect to the new onboarding flow.
* Select various combinations of country/type/structure and verify the requirements component loads correctly in each case.

-------------------


**Post merge**
